### ISSUE TITLE
Old version of tslint used

### ISSUE THREAD

**Issue by tomitrescak** [panuhorsmalahti/gulp-tslint#18]
Hi

Thanks for the plugin! I have a bit of a problem that you do not provide for use of latest language enhancements such as "for of" loop and linting fails. Tslint already supports that, or at least it is not throwing errors. Seems like you are using outdated ts lint?

Following is ok for tslint, but fails with gulp-tslint with use of undeclared variable and duplicate variable

```javascript
var fixtures : string[] = ["siteConfig"];
for (var fixture of fixtures) {
}
```

Thanks!

**Comment by panuhorsmalahti** [comment]
Hmm, gulp-tslint uses the latest version of tslint.

**Comment by panuhorsmalahti** [comment]
I added a test case which doesn't report any errors for me:
https://github.com/panuhorsmalahti/gulp-tslint/commit/e2bc2d570c1420b19f8796cc6fbc95cc59c51c46
Can you try installing gulp-tslint@3.0.1-beta to an empty folder with ``npm install gulp-tslint@3.0.1``, then ``cd test``and ``gulp forof`` and paste the output?

**Comment by tomitrescak** [comment]
That's weird. I did update on gulp tslint and it did have older version of
tslint so I had to manually replace it in node modules.

**Comment by tomitrescak** [comment]
Ok, manually removing tslint from the project and from ~/.npm and reinstalling it solved the issue. The weird part is that npm update did nothing. Thanks for your help.
